JS/fluentRevealNavbar.uc.js

```diff
@@ -135,10 +184,17 @@
       let { gradientSize, lightColor } = this._options;
       let isBookmark =
         el.id === "PlacesChevron" || el.classList.contains("bookmark-item");
-      let area = isBookmark
+      let area = isBookmark 
         ? el
-        : el.querySelector(".toolbarbutton-badge-stack") ||
-          el.querySelector(".toolbarbutton-icon");
+         : el.querySelector(".toolbarbutton-badge-stack") ||
+           el.querySelector(".toolbarbutton-icon");
+      if (!area && el.id == "urlbar-background") area = el;
```

Comment:
```suggestion
      if (el.id == "urlbar-background") area = el;
```

You actually don't need the `!area` anymore